fury/data/fetcher.py

```diff
@@ -349,25 +357,30 @@ async def _fetch_gltf(name, mode):
         )
         return f_names
     else:
-        url = f'{GLTF_DATA_URL}{name}/{mode}'
+        path = f'{name}/{mode}'
+        DATA_DIR = pjoin(dirname(__file__), 'files')
+        with open(pjoin(DATA_DIR, 'KhronosGltfSamples.json'), 'r') as f:
+            models = json.loads(f.read())
 
-        async with aiohttp.ClientSession() as session:
-            request = await _request(session, url)
+        urls = models.get(path, None)
 
-            name = pjoin('glTF', name)
-            name = pjoin(name, mode)
-            folder = pjoin(fury_home, name)
+        if urls is None:
+            raise ValueError(
+                "Model name and mode combination doesn't exist")
 
-            if not os.path.exists(folder):
-                print("Creating new folder ", folder)
-                os.makedirs(folder)
+        path = pjoin(name, mode)
+        path = pjoin('glTF', path)
+        folder = pjoin(fury_home, path)
+        if not os.path.exists(folder):
+            os.makedirs(folder)
 
-            d_urls = [file['download_url'] for file in request]
-            sizes = [file['size'] for file in request]
-            f_names = [url.split('/')[-1] for url in d_urls]
-            f_paths = [pjoin(folder, name) for name in f_names]
-            zip_url = zip(d_urls, f_paths, sizes)
+        d_urls = [file['download_url'] for file in urls]
+        sizes = [file['size'] for file in urls]
+        f_names = [url.split('/')[-1] for url in d_urls]
```

Comment:
I am surprised that this line works on `Windows`.

@ganimtron-10 and @m-agour, can you show me the result (return value) of `fetch_gltf(...)`?

Reply:
@skoudoro, it returns `(['Fox.bin', 'Fox.gltf', 'Texture.png'], 'C:\\Users\\msi\\.fury\\glTF\\Fox\\glTF')`
I checked the folder. It's correct and includes the three files.

Reply:
Thanks for the review @skoudoro. `d_urls` is a list of all download URLs, as parts of the URL are joined by `/`. I'm splitting the URL and getting the last element which is the name of the file.

Reply:
Getting similar path value as given by @m-agour !!